Marlin/macros.h

```diff
@@ -4,6 +4,8 @@
 // Macros for bit masks
 #define BIT(b) (1<<(b))
 #define TEST(n,b) (((n)&BIT(b))!=0)
+#define BITSET(n,b) n |= BIT(b)
+#define BITCLR(n,b) n &= ~BIT(b)
 #define SET_BIT(n,b,value) (n) ^= ((-value)^(n)) & (BIT(b))
```

Comment:
The `sbi()` and `cbi()` macros set and clear a bit in a memory location. These macros will work with any data, including a register, and do not require that the data be a single byte.